src/darwin/Framework/CHIP/templates/helper.js

```diff
@@ -145,6 +145,40 @@ function commandHasRequiredField(command)
   return command.arguments.some(arg => !arg.isOptional);
 }
 
+function enumName(clusterName, enumLabel)
```

Comment:
consider naming objCEnumName since I wonder if this is language specific. I imagine in java or C++ the enums could be namespaced by cluster, so the name is not a concatenation but rather something like `Cluster::Enum` naming or similar.

Also please add a comment with examples on what it does and expected output.

Reply:
> consider naming objCEnumName since I wonder if this is language specific

It is, but this is also a Darwin-specific helper file...  But yes, can do that.